### ISSUE TITLE
MacOS build issue

### ISSUE THREAD

**Issue by JB-Luke** [kronihias/mcfx#6]
Hello,

Nice work, first of all.

I have this issue trying compiling with MacOS Mojave 10.14.6
I got reported 4 errors related to some SDKs of xcode (see attached log file)

What version of xcode are you using? I tried both last available for Mojave and (11.3.1) and for High Sierra (10.1) my cmake went fine but no luck with the build.

I tried in a windows machine but it gives different problems related to SDKs..
[terminal-log.txt](https://github.com/kronihias/mcfx/files/4529506/terminal-log.txt)

**Comment by JB-Luke** [comment]
Trying to building it using Xcode generator instead of unix makefile in cmake solved previous problem.

Now, the code still doesn't compile giving an error realted to VST SDKs (here, like in windows platform), in particular related to 'aeffect.h' and 'aeffectx.h'.

I tried download the SDKs from the Steinberg developers section but I just found a different header files called 'audioeffect.h' and 'audioeffectx.h' which actually gives other problems (attached a screenshot of xcode).
![Screenshot 2020-04-27 at 15 07 07](https://user-images.githubusercontent.com/47320684/80375653-f1b81100-8898-11ea-8f3d-49667cfcc620.png)

Any suggestion? 

I can't find anywhere these SDKs that you mentioned in the code (vstsdk3610_11_06_2018_build_37 or older) and appears that Steinberg has made them gone from any github repos...
